cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
```

Comment:
I think it makes sense to also update the other jobimpls, right? I'll start working on it unless there are any concerns.